ci-scripts/collect-results.sh

```diff
@@ -30,11 +30,15 @@ set -u
 mstart=$(date --utc --date "$(cat benchmark-before)" --iso-8601=seconds)
 mend=$(date --utc --date "$(cat benchmark-after)" --iso-8601=seconds)
 mhost=$(kubectl -n openshift-monitoring get route -l app.kubernetes.io/name=thanos-query -o json | jq --raw-output '.items[0].spec.host')
+mversion=$(sed -n 's/^__version__ = "\(.*\)"/\1/p' scenarios/$(cat benchmark-scenario).py)
```

Comment:
Why cannot you just use th `SCENARIO` env variable directly?
```suggestion
mversion=$(sed -n 's/^__version__ = "\(.*\)"/\1/p' scenarios/${SCENARIO}.py)
```

Reply:
We only have that variable in `ci-scripts/test.sh`. Maybe if this becomes secret in Vault, we can take if from that file.

Reply:
Actually, the `SCENARIO` env variable along with [others](https://github.com/openshift/release/blob/master/ci-operator/config/redhat-performance/backstage-performance/redhat-performance-backstage-performance-main.yaml#L50) is set by the CI and in the `ci-scripts/test.sh` is just overriden temporarily and hard-coded to the only rhdh scenario we currently have. So, you are correct that to make it work in the `collect-results.sh` we should override it similarly as in `test.sh` (for now).

Reply:
Let's do that another time